source/index.html.md

```diff
@@ -186,13 +186,57 @@ This endpoint finds  customer record by its id
 
 ## Create a subscription record under a business account
 
-> **Example Subscription body**
+### Subscription policy
+
+> **Example Subscription body with SUBSCRIPTION policy**
```

Comment:
This should be `Example Subscription body for a tier whose policy value is SUBSCRIPTION`